catalyst_voices/lib/widgets/common/columns_row.dart

```diff
@@ -0,0 +1,88 @@
+import 'package:catalyst_voices/widgets/seed_phrase/seed_phrases_completer.dart';
+import 'package:catalyst_voices/widgets/seed_phrase/seed_phrases_picker.dart';
+import 'package:catalyst_voices/widgets/seed_phrase/seed_phrases_viewer.dart';
+import 'package:collection/collection.dart';
+import 'package:flutter/material.dart';
+
+/// A widget that arranges children into columns within a row.
+///
+/// This widget divides its children into columns based on the [columnsCount]
+/// property. Each column is then laid out vertically with optional spacing
+/// between children.
+///
+/// The [mainAxisSpacing] property controls the horizontal spacing between
+/// columns, while the [crossAxisSpacing] property controls the vertical
+/// spacing within each column.
+///
+/// See following widgets for examples of usage.
+///   - [SeedPhrasesViewer], [SeedPhrasesPicker] and [SeedPhrasesCompleter]
+class ColumnsRow extends StatelessWidget {
```

Comment:
First thought, was it not possible to use `Wrap` for this? Not sure whether it's configurable enough to achieve what we wanted. If it's possible to configure it then I think that's better than our own widget, if not - our widget is perfect

Reply:
Sadly `Wrap` does not allow us to specify number or columns, which is required here.

I wanted to use it at first too :(

Reply:
And `GridView` is scrollable and is forcing `aspectRatio` on child's instead of letting us decide.